pkg/k8s/deployments/crud.go

```diff
@@ -95,9 +95,6 @@ func GetTranslations(dev *model.Dev, d *appsv1.Deployment, c *kubernetes.Clients
 	for _, s := range dev.Services {
 		d, err := Get(s, dev.Namespace, c)
 		if err != nil {
-			if errors.IsNotFound(err) {
-				continue
-			}
 			return nil, err
```

Comment:
Will this be a user friendly error or the k8s one?

Reply:
Right it says the deployment that does not exist